File: PHP/Program/Dpr.php
<?php
class Dpr{

    // PRIVATE ATRIBUT
    private $id = "";
    private $nama = "";
    private $bidang = "";
    private $partai = "";
    private $foto = "";

    // CONSTRUCT
    function __construct($id = "", $nama = "", $bidang = "", $partai = "", $foto = ""){
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = $foto;
    }

    // PUBLIC METHOD (SETTER & GETTER)
    function setid($id = ""){
        $this->id = $id;
    }

    function getid(){
        return $this->id;
    }

    function setnama($nama = ""){
        $this->nama = $nama;
    }

    function getnama(){
        return $this->nama;
    }

    function setbidang($bidang = ""){
        $this->bidang = $bidang;
    }

    function getbidang(){
        return $this->bidang;
    }

    function setpartai($partai = ""){
        $this->partai = $partai;
    }

    function getpartai(){
        return $this->partai;
    }

    function setFoto($foto = ""){
        $this->foto = $foto;
    }

    function getFoto(){
        return $this->foto;
    }
    
    // DESTRUCTOR
    function __destruct(){
    }
}

?>

File: Python/Program/Dpr.py
class Dpr:
    def __init__(self, id="", nama="", nama_bidang="", nama_partai=""):
        # Inisialisasi objek Dpr dengan atribut yang diberikan
        self.id = id
        self.nama = nama
        self.nama_bidang = nama_bidang
        self.nama_partai = nama_partai

    def get_id(self):
        # Mengembalikan id anggota DPR
        return self.id

    def set_id(self, id):
        # Mengatur id anggota DPR
        self.id = id

    def get_nama(self):
        # Mengembalikan nama anggota DPR
        return self.nama

    def set_nama(self, nama):
        # Mengatur nama anggota DPR
        self.nama = nama

    def get_nama_bidang(self):
        # Mengembalikan nama bidang kerja anggota DPR
        return self.nama_bidang

    def set_nama_bidang(self, nama_bidang):
        # Mengatur nama bidang kerja anggota DPR
        self.nama_bidang = nama_bidang

    def get_nama_partai(self):
        # Mengembalikan nama partai anggota DPR
        return self.nama_partai

    def set_nama_partai(self, nama_partai):
        # Mengatur nama partai anggota DPR
        self.nama_partai = nama_partai
